### ISSUE TITLE
[BUG] AttributeError: 'NoneType' object has no attribute 'profile'

### ISSUE THREAD

**Issue by datascientistlin** [databrickslabs/dbx#369]
## Expected Behavior
`dbx launch` completes for job run

## Current Behavior
`dbx launch` failed with error 

> AttributeError: 'NoneType' object has no attribute 'profile'

## Steps to Reproduce (for bugs)
Following tutorials from https://docs.microsoft.com/en-us/azure/databricks/dev-tools/ide-how-to to set up IDE and interact with Datbricks
Error occured in the [Run with dbx](https://docs.microsoft.com/en-us/azure/databricks/dev-tools/ide-how-to) step

## Context
I have set up my connection profile (dev). The `environments` in project.json and deployment.yml have been adjusted to my environemnt dev as well. I then successfully deployed the app with command `dbx deploy --environment=dev`. When executing `dbx launch --job=covid_analysis_etl_integ`, the error occured. The full error message is below:
```
[dbx][2022-08-04 11:31:36.541] Launching job covid_analysis_etl_prod on environment default
[dbx][2022-08-04 11:31:36.544] Using profile provided from the project file
Traceback (most recent call last):
  File "/Users/di_lin/opt/anaconda3/envs/erebus/bin/dbx", line 8, in <module>
    sys.exit(cli())
  File "/Users/di_lin/opt/anaconda3/envs/erebus/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/Users/di_lin/opt/anaconda3/envs/erebus/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/Users/di_lin/opt/anaconda3/envs/erebus/lib/python3.9/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/di_lin/opt/anaconda3/envs/erebus/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/di_lin/opt/anaconda3/envs/erebus/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/Users/di_lin/opt/anaconda3/envs/erebus/lib/python3.9/site-packages/dbx/commands/launch.py", line 137, in launch
    api_client = prepare_environment(environment)
  File "/Users/di_lin/opt/anaconda3/envs/erebus/lib/python3.9/site-packages/dbx/utils/common.py", line 35, in prepare_environment
    transfer_profile_name(info)
  File "/Users/di_lin/opt/anaconda3/envs/erebus/lib/python3.9/site-packages/dbx/utils/common.py", line 28, in transfer_profile_name
    os.environ[ProfileEnvConfigProvider.DBX_PROFILE_ENV] = info.profile
AttributeError: 'NoneType' object has no attribute 'profile'
```

## Your Environment

* dbx version used: 0.6.8
* Databricks Runtime version:

**Comment by renardeinside** [comment]
hi @datascientistlin , 
could you please check what happens if you add the env parameter to your launch? 
e.g.:
```
dbx deploy --environment=dev
dbx launch --environment=dev --job=...
```
?

**Comment by datascientistlin** [comment]
Thanks @renardeinside . This works perfect! How can I foget about the `environment` argument...

A follow-up question, do you know if there is any instruction for setting the CICD flow in Azure DevOps? I could see the GitHub Action approach in the tutorial, but it would be great if there is something pertaining to Azure DevOps as well.

Thank you.

**Comment by renardeinside** [comment]
Anyways this is considered as a bug, it should fail before reaching this point, so thanks for reporting!

As for Azure DevOps - you can take a look at the `dbx init` and the Azure DevOps template provided there.

**Comment by datascientistlin** [comment]
Thanks a lot for the advise. Could you please help me understand which `dbx initi` you are referring to?

**Comment by renardeinside** [comment]
I mean this one - `dbx init`,the doc is [here](https://dbx.readthedocs.io/en/latest/cli.html#dbx-init).

**Comment by datascientistlin** [comment]
Got it! Thanks a lot

**Comment by renardeinside** [comment]
resolved in linked PR, released in v0.6.10, thanks a lot for raising this one @datascientistlin !
